src/Core/Defines.h

```diff
@@ -136,5 +136,6 @@ static constexpr auto QUERY_PROFILER_DEFAULT_SAMPLE_RATE_NS = 0;
 #endif
 
 static constexpr auto DEFAULT_REMOVE_SHARED_RECURSIVE_FILE_LIMIT = 1000uz;
+static constexpr auto DBMS_DEFAULT_MAX_RETRIES = 100;
```

Comment:
I'd not put it in `Defines.h`, retries limit is so specific to the concrete use case, it seems incorrect to put it here.